stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py

```diff
@@ -91,10 +96,15 @@ def delete_item(
         """Delete item."""
         base_url = str(kwargs["request"].base_url)
         with self.session.writer.context_session() as session:
+            query = session.query(self.item_table).filter(
```

Comment:
This definition of `query` seems to be overridden by the next line. Perhaps use `query` instead of `session.query(...)` in the next line.

Reply:
True thanks